src/components/principal/Principal.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="principal">
+    <h1>Principal</h1>
+    <div class="units-container">
+      <div class="unit-wrapper" v-for="unit in units" v-bind:key="unit.id">
+        <Unit :unit="unit"></Unit>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import Unit from './unit/Unit.vue'
+
+export default {
+  name: 'Principal',
+  components: {
+    Unit,
```

Comment:
I don't know if this ',' is needed after last component(in this case we have only 1)